ui.frontend/src/view/FormFieldBase.js

```diff
@@ -78,6 +93,51 @@ export default class FormFieldBase extends FormField {
         }
         this._updateVisible(state.visible)
         this._updateEnable(state.visible)
+        this._initializeHelpContent();
+    }
+
+    _initializeHelpContent() {
+        // Initializing Tooltip
+        if (!this._isTooltipAlwaysVisible()) {
+            const state = this._model.getState();
+            let txt =  '';
+            if (state) {
+                // Short description is saved as rich text. Hence, removing html tags for title attribute
+                txt = state.tooltip ? state.tooltip.replace(/<\/?[^>]+(>|$)/g, "") : '';
+            }
+            const widget = this.getWidget();
+            if (widget) {
+                widget.setAttribute('title', txt);
+            }
```

Comment:
also you can directly use this.widget as it has been initialized in the constructor